src/modules/m_cgiirc.cpp

```diff
@@ -338,12 +338,12 @@ class ModuleCgiIRC
 			}
 		}
 
-		// The host configuration was valid so we can apply it.
-		hosts.swap(identhosts);
-		cmd.hosts.swap(webirchosts);
-
 		// Do we send an oper notice when a m_cgiirc client has their IP changed?
 		cmd.notify = ServerInstance->Config->ConfValue("cgiirc")->getBool("opernotice", true);
+
+		// The configuration was valid so we can apply it.
+		hosts.swap(identhosts);
```

Comment:
This was fine as it was before?

Reply:
yeah, the move was done when I thought `getBool()` and similar might throw in the future